org.uqbar.project.wollok.typeSystem/src/org/uqbar/project/wollok/typesystem/GenericType.xtend

```diff
@@ -15,4 +26,16 @@ class GenericType extends ClassBasedWollokType {
 		this.typeParameterNames = typeParameterNames
 	}
 	
+	def instance() {
+		instance(typeParameterNames.toInvertedMap[TypeVariable.synthetic])
+	}
+	
+	def instance(Map<String, ITypeVariable> typeParameters) {
+		new GenericTypeInstance(this, typeParameters)
+	} 
+	
+	def toString(GenericTypeInstance instance) {
+		// TODO This is partial implementation to avoid modifying all the tests right now.
+		toString 
```

Comment:
Acá deberíamos cambiar que cuando hoy se está devolviendo el tipo "List", en realidad es lista de algo, por ejemplo debería ser "List<String>" o "List of String". El ts tiene es info internamente, pero los tests son anteriores y tiran asserts por List, entonces corregirlo es fácil pero hay que cambiar todos los tests.

Reply:
Yo me puedo encargar de corregir los tests, eso no es problema.